<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-LVNM1PD" failures="0" tests="2" name="testNG_PACKAGE.CreateLead_TS_POM_02_TestNG" time="0.039" errors="1" timestamp="2023-01-11T16:27:35 IST" skipped="1">
  <testcase classname="testNG_PACKAGE.CreateLead_TS_POM_02_TestNG" name="createLead" time="0.000">
    <skipped/>
  </testcase> <!-- createLead -->
  <system-out/>
  <testcase classname="testNG_PACKAGE.CreateLead_TS_POM_02_TestNG" name="configDB" time="0.039">
    <error message="No suitable driver found for http://rmgtestingserver:8888/" type="java.sql.SQLException">
      <![CDATA[java.sql.SQLException: No suitable driver found for http://rmgtestingserver:8888/
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:706)
at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:229)
at com.crm.GenericUtility.DatabaseUtility.connectionToDB(DatabaseUtility.java:21)
at com.crm.GenericUtility.BaseClass.configDB(BaseClass.java:28)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:317)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- configDB -->
  <system-out/>
</testsuite> <!-- testNG_PACKAGE.CreateLead_TS_POM_02_TestNG -->
